modin/pandas/test/test_groupby.py

```diff
@@ -2715,7 +2721,7 @@ def test_skew_corner_cases():
     "by",
     [
         pandas.Grouper(key="time_stamp", freq="3D"),
-        [pandas.Grouper(key="time_stamp", freq="1M"), "count"],
+        [pandas.Grouper(key="time_stamp", freq="1ME"), "count"],
```

Comment:
Is this correct? What does `ME` mean?

Reply:
Looks like `MonthEnd` https://pandas.pydata.org/pandas-docs/version/2.2.0/whatsnew/v2.2.0.html#deprecate-aliases-m-q-y-etc-in-favour-of-me-qe-ye-etc-for-offsets

Reply:
I see